src/PaymentSuite/GestpayBundle/Tests/Unit/Services/GestpayEncrypterTest.php

```diff
@@ -63,15 +64,15 @@ public function testEncrypt()
         $paymentBridge = new DummyPaymentBridge();
         $currencyResolver = new GestpayCurrencyResolver($paymentBridge);
         $transactionIdAssembler = new GestpayTransactionIdAssembler($paymentBridge);
+        $settingsProvider = new GestpaySettingsProviderDefault($shopLogin, $apiKey);
 
         $encrypter = new GestpayEncrypter(
             $encryptClient->reveal(),
             $paymentBridge,
             $currencyResolver,
             $transactionIdAssembler,
-            $sandbox,
-            $shopLogin,
-            $apiKey
+            $settingsProvider,
+            $sandbox
```

Comment:
I've decided to leave sandbox configuration out of settings provider for every payment method which have a "sandbox" mode. This is because I've considered that sandbox mode is independent from merchant authentication data, and it is related to API endpoints which hardly change in time. Additionally, if you think of it, you usually enable/disable sandbox by app environment dev/prod, and it may entail to change auth settings for some payment methods (but not for many others). So I decided to be consistent with this idea for every method and leave this config as it was.